webapp/src/scriptmanager.tsx

```diff
@@ -300,6 +308,10 @@ export class ScriptManagerDialog extends data.Component<ScriptManagerDialogProps
         return sortingFunction;
     }
 
+    private getId(scr: pxt.workspace.Header) {
```

Comment:
I think this might have broken the Delete button. See https://github.com/microsoft/pxt-arcade/issues/3676